lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -164,7 +165,10 @@ namespace Js
             }
         }
     };
+#endif
 
+// Defining Finalizable wrappers for Intl data
+#ifdef INTL_WINGLOB
```

Comment:
The above endif is for INTL_WINGLOB also. Why have two separate blocks?

Reply:
I think I did this yesterday while taking notes on all of the different classes, where they all lived, etc so that the elif of this if had the same corresponding functionality, in that they enabled auto-pointers for their respective backing library (I think it was specifically because I wanted to add that comment).

TL;DR I can revert it, as it was an artifact of re-organizing things so I could understand them better. Now that I understand them, if you don't think there is much value added, I can remove it.